ResearchKit/Common/ORKInstructionStep.m

```diff
@@ -77,6 +79,7 @@ + (BOOL)supportsSecureCoding {
 - (instancetype)copyWithZone:(NSZone *)zone {
     ORKInstructionStep *step = [super copyWithZone:zone];
     step.detailText = self.detailText;
```

Comment:
Not part of your changes, but can you also update this to be [self.detailText copy]